process/smartContract/processorV2/processV2.go

```diff
@@ -2257,12 +2268,18 @@ func createBaseSCR(
 	result.CallType = vmData.DirectCall
 	setOriginalTxHash(result, txHash, tx)
 
-	relayedTx, isRelayed := isRelayedTx(tx)
+	relayedTx, isRelayed := isRelayedSCR(tx)
```

Comment:
duplicated code here and on createAsnyCallBackSCRFromVMOutput. 

Make a new function and use it in both places.

duplicated from 2271 - 2282